source/managing-your-data/updating-a-dataset-schema.md

```diff
@@ -0,0 +1,75 @@
+# Updating a Dataset Schema
+
+There are various reasons you may have for modifying a dataset schema. Here are some possible reasons:
+
+- Enrich your dataset a new property
+- Remove a deprecated property
+- Change a property type
+- Require values for a property
+- Forbid null values for a property
+- Index a property
+- SmartLink a property
```

Comment:
Property instead of 'field' seems odd? But if that's our tone/standard, that's fine.

Reply:
Regarding 'property' vs 'field', we refer to them as 'properties' in the UI. For example, the schema editor shows these in a **Properties** table and has an option to **Add a property**. So I primarily use the term 'property'.

In the API reference, we use the term 'attribute' because in many cases we're passing around JavaScript objects, where the properties are referred to as attributes.

... so I'm trying to stick to those two. :-)

Reply:
Makes sense!